tests/utils/vmgroup/vmgrouplifecycle.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This util is going to hold various helper methods related to vmgroup to be consumed by testcases.
+// vmgroup creation, deletion is supported as of now.
+
+
+package vmgroup
```

Comment:
vmgroup creation/deletion should be considered as part of AdminCLI. 

I would suggest to place at /utils/admincli

Following is the highlevel structure

```
tests
   |-e2e
   |-consts
   |-utils
      |-admincli ( if you want you may create subfolder here)
      |-dockercli
      |-govc
      |-misc
      |- ...
```